ui/src/writeData/components/WriteDataSection.tsx

```diff
@@ -0,0 +1,72 @@
+// Libraries
+import React, {FC, useContext} from 'react'
+import {sortBy} from 'lodash'
+
+// Contexts
+import {WriteDataSearchContext} from 'src/writeData/containers/WriteDataPage'
+
+// Components
+import {
+  SquareGrid,
+  ComponentSize,
+  Heading,
+  HeadingElement,
+  FontWeight,
+} from '@influxdata/clockface'
+import WriteDataItem from 'src/writeData/components/WriteDataItem'
+
+// Constants
+import {doesItemMatchSearchTerm} from 'src/writeData/constants'
+
+// Types
+import {WriteDataSection} from 'src/writeData/constants'
+
+const WriteDataSection: FC<Omit<WriteDataSection, 'featureFlag'>> = ({
+  id,
+  name,
+  description,
+  items,
+}) => {
+  const {searchTerm} = useContext(WriteDataSearchContext)
+
+  const filteredItems = items.filter(item =>
+    doesItemMatchSearchTerm(item.name, searchTerm)
+  )
+
+  const sortedItems = sortBy(filteredItems, item => item.name.toLowerCase())
```

Comment:
`filteredItems.sort((a,b) => a.name.toLowerCase().localeCompare(b.name.toLowerCase()))`

now get that lodash nonsense outta here